### PR TITLE
adding new oidc-equality-helper.service

### PR DESCRIPTION
**Pull request by damienbod**

### CODE REVIEW

File: src/modules/auth.configuration.ts [hunk 1]
```diff
@@ -188,39 +192,44 @@ export class AuthConfiguration {
...
 
     get trigger_authorization_result_event(): boolean {
         if (this.openIDImplicitFlowConfiguration) {
-            return this.openIDImplicitFlowConfiguration.trigger_authorization_result_event;
+            return this.openIDImplicitFlowConfiguration
+                .trigger_authorization_result_event;
         }
 
         return this.defaultConfig.trigger_authorization_result_event;
     }
```
Comment (damienbod) [inline src/modules/auth.configuration.ts]: can you leave this as it was?

File: src/modules/auth.configuration.ts [hunk 2]
```diff
@@ -188,39 +192,44 @@ export class AuthConfiguration {
...
-    get log_console_warning_active(): boolean {
+    get isLogLevelWarningEnabled(): boolean {
         if (this.openIDImplicitFlowConfiguration) {
-            return this.openIDImplicitFlowConfiguration.log_console_warning_active;
+            return this.openIDImplicitFlowConfiguration
+                .log_console_warning_active;
         }
 
         return this.defaultConfig.log_console_warning_active;
     }
```
Comment (damienbod) [inline src/modules/auth.configuration.ts]: can you leave this as it was?

File: src/services/oidc-data.service.ts [hunk 3]
```diff
@@ -11,7 +11,7 @@ export class OidcDataService {
         headers = headers.set('Accept', 'application/json');
 
         return this.httpClient.get<T>(url, {
```
Comment (damienbod) [inline src/services/oidc-data.service.ts]: comma too many?

File: src/services/oidc-data.service.ts [hunk 4]
```diff
@@ -24,7 +24,7 @@ export class OidcDataService {
         );
 
         return this.httpClient.get<T>(url, {
```
Comment (damienbod) [inline src/services/oidc-data.service.ts]: comma too many?

File: src/services/oidc-data.service.ts [hunk 5]
```diff
@@ -33,7 +33,7 @@ export class OidcDataService {
         headers = headers.set('Accept', 'application/json');
 
         return this.httpClient.get<T>(url, {
-            headers: headers
+            headers: headers,
```
Comment (damienbod) [inline src/services/oidc-data.service.ts]: comma too many?

File: src/services/oidc-data.service.ts (hunk 3 above)
Comment (FabianGosebrink) [reply src/services/oidc-data.service.ts]: no because of git code styling --> use trailing commas

File: src/services/oidc-data.service.ts (hunk 4 above)
Comment (FabianGosebrink) [reply src/services/oidc-data.service.ts]: no because of git code styling --> use trailing commas

File: src/services/oidc-data.service.ts (hunk 5 above)
Comment (FabianGosebrink) [reply src/services/oidc-data.service.ts]: no because of git code styling --> use trailing commas

**Pull request merged**